HW04/anastasia_evsihneyeva/lib/homework.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+class Homework
+  def initialize(data, mentor)
+    @title = data[:title]
+    @description = data[:desription]
+    @student = data[:student]
+    @mentor = mentor
+    @status = 'new'
+    @answers = []
+    @to_check = []
+  end
+
+  def add_answer!(answer)
+    @answers << answer
```

Comment:
add whatever you are using to attr_reader
```suggestion
    answers << answer
```

Reply:
But how can I do list attr_reader? I can do it only with symbols and if I try to use << with symbol appears an error...